examples/check-full.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: r-lib/actions/setup-r@v1
```

Comment:
The examples should be using v1, not master, but the versions in `.github/workflows` should use master.

Running the makefile in the root directory should copy the files automatically and make this change.

Reply:
Oh right, I didn't see your comment above, disregard this.

Reply:
This reason could be enough to not merge the PR. Up to you